Kernel/Syscalls/execve.cpp

```diff
@@ -114,6 +114,13 @@ static ErrorOr<FlatPtr> make_userspace_context_for_main_thread([[maybe_unused]]
             push_string_on_new_stack(value.optional_string);
             value.auxv.a_un.a_ptr = (void*)new_sp;
         }
+        if (value.auxv.a_type == ELF::AuxiliaryValue::Random) {
+            char* buffer;
+            auto random_bytes = TRY(KString::try_create_uninitialized(16, buffer));
```

Comment:
I don't think we need to allocate a KString for a fixed size 16 byte buffer, do we? The old code with the exactly 16 byte StringView into a stack buffer should work just fine

Reply:
Oh yeah, not sure why I over-complicated it so much.